qiita_db/support_files/patches/19.sql

```diff
@@ -0,0 +1,34 @@
+-- March 19, 2015
+-- Removes the status from the study and adds it to the processed data table.
+
+-- Modify the study_status_table so it becomes the processed_data_status table
+ALTER TABLE qiita.study_status RENAME TO processed_data_status;
+ALTER TABLE qiita.processed_data_status RENAME COLUMN study_status_id TO processed_data_status_id;
+ALTER TABLE qiita.processed_data_status RENAME COLUMN status TO processed_data_status;
```

Comment:
Sorry, we had mixed set ups and I thought that this one was the one that we wanted to standardize... Removing that line.